code/src/Components/Result.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+export const Result = ({
+  adjective,
+  foodNoun,
+  ingVerb,
+  clothingNoun,
+  celebrity,
+  jobSector,
+  happyAdjective,
+  bodyPart,
+  infinitiveVerb,
+  descriptiveAdjective,
+  introPage
+}) => {
+  return (
```

Comment:
Love the text in the result section!
Perhaps the input that the user has been given can be highlighted or in bold so that they stand out a bit.